.travis.yml

```diff
@@ -21,6 +21,7 @@ matrix:
     fast_finish: true
 
 before_script:
+  - composer install
   - phpenv config-add ci/travis.php.ini
   - composer remove --dev spryker/git-hook
```

Comment:
why do we need this? we should remove the whole thing for the repo IMO.

Reply:
Composer couldn' remove spryker/git-hook without install command

Reply:
i would also remove the whole git-hook thing from core modules.

Reply:
Got it